src/redirects.js

```diff
@@ -162,6 +162,12 @@ export function updateDatasetUrl(originalUrl) {
     ['/monkeypox/hmpxv1', '/mpox/clade-IIb'],
     ['/monkeypox/hmpxv1/big', '/mpox/lineage-B.1'],
     ["/monkeypox", "/mpox"],
+    /**
+     * We shifted avian-flu and seasonal-flu to top level to stop nesting them
+     * both under flu/ so that avian-flu is more discoverable
+     */
+    ['/flu/avian(.*)', '/avian-flu(.*)'],
+    ['/flu/seasonal(.*)', '/seasonal-flu(.*)'],
```

Comment:
I'm including this mostly to demonstrate how we would redirect the flu paths.
I can drop it if we need to coordinate change with https://github.com/nextstrain/nextstrain.org/pull/811

Reply:
Yeah - nice to see how well these work! I'd say drop this commit from this PR and we can add it to a subsequent PR depending on the migration path we choose to take.

Reply:
Dropped https://github.com/nextstrain/nextstrain.org/commit/3e8319440a77b1ab7206f34c3fa7085546e33c21